src/main/scala/org/opennetworkinsight/ProxyPostLDA.scala

```diff
@@ -0,0 +1,126 @@
+package org.opennetworkinsight
+
+import org.apache.log4j.{Level, Logger => ApacheLogger}
+import org.apache.spark.sql.SQLContext
+import org.apache.spark.storage.StorageLevel
+import org.apache.spark.SparkContext
+import org.slf4j.{Logger, LoggerFactory}
+
+/**
+  *
+  */
+object ProxyPostLDA {
+
+  def getColumnNames(input: Array[String], sep: Char = ','): scala.collection.mutable.Map[String, Int] = {
+    val columns = scala.collection.mutable.Map[String, Int]()
+    val header = input.zipWithIndex
+    header.foreach(tuple => columns(tuple._1) = tuple._2)
+    columns
+  }
+
+  def proxyPostLDA(inputPath: String, resultsFilePath: String, threshold: Double, documentResults: Array[String],
+                 wordResults: Array[String], sc: SparkContext, sqlContext: SQLContext, logger: Logger) = {
+
+    val logger = LoggerFactory.getLogger(this.getClass)
+    ApacheLogger.getLogger("org").setLevel(Level.OFF)
+    ApacheLogger.getLogger("akka").setLevel(Level.OFF)
```

Comment:
I don't think we need this code - either. Logger is being passed to proxyPostLDA. 
Also apacheLogger doesn't need to be re-configured/turned off